packages/faust-nx/package.json

```diff
@@ -6,18 +6,20 @@
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
   "peerDependencies": {
+    "@apollo/client": ">=3.6.6",
     "next": ">=12.1.6",
     "react": ">=17.0.2",
-    "react-dom": ">=17.0.2",
-    "@apollo/client": ">=3.6.6"
+    "react-dom": ">=17.0.2"
   },
   "devDependencies": {
+    "@apollo/client": "^3.6.6",
+    "@types/configstore": "^6.0.0",
     "next": "^12.1.6",
     "react": "^17.0.2",
-    "react-dom": "^17.0.2",
-    "@apollo/client": "^3.6.6"
+    "react-dom": "^17.0.2"
   },
   "dependencies": {
+    "configstore": "^6.0.0",
     "deepmerge": "^4.2.2",
     "graphql": "^16.5.0",
     "lodash": "^4.17.21"
```

Comment:
I might be incorrect on this in the context of SSR, but I'd consider moving `react`, `react-dom`, & `@apollo/client` to `dependencies` as they are used client side.

In addition, should our peerDependencies contain packages FaustNX already has defined as `dependencies` 
 or `devDependencies`?

Reply:
Makes sense thanks @blakewilson!